### PR TITLE
refactor(mini-rx-store): use map instead of mapTo

### PR DESCRIPTION
**Pull request by carlos-ds**
See https://rxjs.dev/api/operators/mapTo : `mapTo` will be removed in RxJS v9. Use `map` instead.

### CODE REVIEW

**Comment by spierala** [comment]
Looks nice!

When do you want to merge this?

I guess there could be merge conflicts with the mini-rx-store refactor code.

My suggestion would be to wait till all the big stuff is merged into release 2024.1.

**Review by spierala — APPROVED** [review]

**Comment by carlos-ds** [comment]
Sure, that's fine with me. We'll wait until the other stuff is merged into `release-2024.1`.

**Comment by carlos-ds** [comment]
@spierala I resolved the merge conflicts with release-2024.1 so should be good to merge as well.

**Pull request merged**
